src/black/mode.py

```diff
@@ -163,7 +162,9 @@ def __contains__(self, feature: Preview) -> bool:
         """
         if feature is Preview.string_processing:
             return self.preview or self.experimental_string_processing
-        return self.preview
+        # TODO: Remove type ignore comment once preview contains more features
+        #  than just ESP
+        return self.preview  # type: ignore
```

Comment:
Hmmm? what's up with mypy?

Reply:
It thinks this code is unreachable because there's only one enum member.

Reply:
Yeah ^